marklogic-data-hub-central/ui/src/components/sidebar-footer/sidebar-footer.tsx

```diff
@@ -53,10 +53,10 @@ const SidebarFooter: React.FC = () => {
     <div>
       <HCDivider style={{"backgroundColor": "#CCCCCC", "height": "1px", "opacity": "0.5", "margin": "10px 0px 0px 0px"}} />
       <div className={styles.facetFooter}>
-        <Button className={styles.button} aria-label="clear-facets-button" disabled={searchOptions.tileId === "explore" ?
+        <HCButton variant="outline-light" className={styles.button} aria-label="clear-facets-button" disabled={searchOptions.tileId === "explore" ?
           (Object.keys(searchOptions.selectedFacets).length === 0 && Object.keys(greyedOptions.selectedFacets).length === 0)
-          : (Object.keys(monitorOptions.selectedFacets).length === 0 && Object.keys(monitorGreyedOptions.selectedFacets).length === 0)} onClick={searchOptions.tileId === "explore" ? () => clearAllFacets() : () => clearAllMonitorFacets()}>Clear All Facets</Button>
-        <Button className={styles.button} aria-label="apply-facets-button" disabled={searchOptions.tileId === "explore" ? Object.keys(greyedOptions.selectedFacets).length === 0 : Object.keys(monitorGreyedOptions.selectedFacets).length === 0} onClick={searchOptions.tileId === "explore" ? () => applyFacets() : () => applyMonitorFacets()} type="primary" >Apply Facets</Button>
+          : (Object.keys(monitorOptions.selectedFacets).length === 0 && Object.keys(monitorGreyedOptions.selectedFacets).length === 0)} onClick={searchOptions.tileId === "explore" ? () => clearAllFacets() : () => clearAllMonitorFacets()}>Clear All Facets</HCButton>
```

Comment:
"Clear All Facets" button label should be one row. @jbelonoj do you agree?
<img width="322" alt="clear-all-facets" src="https://user-images.githubusercontent.com/477757/132907126-3e794492-74e7-4fb5-abb0-1fe1ff0abc6e.png">
This occurs in the Explore and Monitor views.

Reply:
Yes, please. Thanks! @wooldridge

Reply:
@wooldridge  @jbelonoj this is the clear buttom
![image](https://user-images.githubusercontent.com/88790827/133124574-2aa8e28f-36c7-4fd8-809d-738c455db1ff.png)